### PR TITLE
2024 edition

### PR DESCRIPTION
**Pull request by StanBoyet**
- Add new 2024 content's edition
- Remove un-referenced pages

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -4,78 +4,47 @@ title: "FOSDEM &lt;3 Ruby"
...
-Ruby devroom not only would the Ruby community become more plugged into the
-FOSDEM experience, but FOSDEM would be hosting Ruby-oriented content that is
-both unique and very technical. Considering the broad interest in dynamic and
-interpreted languages, the Ruby devroom will also provide compelling technical
-content interesting to members of the Python, Perl, JavaScript, Smalltalk, LLVM
-and Java communities. There is quite some interest in the topic, as the
-overcrowded Java dev room if  [Charles Oliver
-Nutter's](https://github.com/headius) talk on JRuby in the FOSDEM 2014 Free
-Java devroom is any indication.
+Join us at the Ruby Devroom at FOSDEM 2023 for a celebration of all things Ruby!
```
Comment (rtyler) [inline index.md:9]:
```suggestion
Join us at the Ruby Devroom at FOSDEM 2024 for a celebration of all things Ruby! 
```

**Review by rtyler — COMMENTED** [review]
Exciting!

**Comment by heliocola** [comment]
@StanBoyet : did you run `bundle install` and run the site on your local machine?
On my M2 it fails with the current state of Gemfile.lock but it does work after a `bundle update`.

File: index.md [hunk 2]
```diff
@@ -4,78 +4,47 @@ title: "FOSDEM &lt;3 Ruby"
...
+* A short bio and contact information
+* Links to related websites / blogs etc
+* Talk duration
 
-## How could you support the devroom?
+## Requirements
 
-If you'd like to help the Ruby devroom to welcome speakers from all around the
-world, [you do it via sponsorship](sponsorship.html).
+The conference language is English. All content must relate to Free and Open Source Software. By participating in the event you agree to the publication of your recordings, slides and other content provided under the same licence as all FOSDEM content (CC-BY). By participating in FOSDEM you are required to adhere the Code of Conduct: https://fosdem.org/2024/practical/conduct/
```
Comment (heliocola) [inline index.md:47]:
```suggestion
The conference language is English. All content must relate to Free and Open Source Software. By participating in the event you agree to the publication of your recordings, slides and other content provided under the same licence as all FOSDEM content (CC-BY). By participating in FOSDEM you are required to adhere the [Code of Conduct](https://fosdem.org/2024/practical/conduct/)
```

File: index.md [hunk 3]
```diff
@@ -4,78 +4,47 @@ title: "FOSDEM &lt;3 Ruby"
...
+Important dates:
+- 1st December 2023: Submission Deadline 
+- 8th December 2023: Acceptance Notification 
+- 15th December 2023: Final Schedule Announcement
 
-{% include committee.md %}
+## Submit a talk
 
-## Code of Conduct
+Head to the FOSDEM 2024 pretalx
```
Comment (heliocola) [inline index.md:33]: Consider removing the new lines and use the link to CFP as `[https://pretalx.fosdem.org/fosdem-2024/cfp](https://pretalx.fosdem.org/fosdem-2024/cfp)` or something similar.

Maybe something like:
`Head to [FOSDEM 2024 Pretalx website](https://pretalx.fosdem.org/fosdem-2024/cfp)`

Comment (StanBoyet) [reply index.md:33]:
```suggestion
Head to the [FOSDEM 2024 Pretalx website](https://pretalx.fosdem.org/fosdem-2024/cfp)
```

File: index.md [hunk 4]
```diff
@@ -4,78 +4,47 @@ title: "FOSDEM &lt;3 Ruby"
...
+- 1st December 2023: Submission Deadline 
+- 8th December 2023: Acceptance Notification 
+- 15th December 2023: Final Schedule Announcement
 
-{% include committee.md %}
+## Submit a talk
 
-## Code of Conduct
+Head to the [FOSDEM 2024 Pretalx website](https://pretalx.fosdem.org/fosdem-2024/cfp)
+(https://pretalx.fosdem.org/fosdem-2024/cfp) website. This has
```
Comment (StanBoyet) [inline index.md:34]:
```suggestion
Head to the [FOSDEM 2024 Pretalx website](https://pretalx.fosdem.org/fosdem-2024/cfp) website. This has
```

File: index.md [hunk 5]
```diff
@@ -4,78 +4,46 @@ title: "FOSDEM &lt;3 Ruby"
...
+Important dates:
+- 1st December 2023: Submission Deadline 
+- 8th December 2023: Acceptance Notification 
+- 15th December 2023: Final Schedule Announcement
 
-{% include committee.md %}
+## Submit a talk
 
-## Code of Conduct
+Head to the [FOSDEM 2024 Pretalx website](https://pretalx.fosdem.org/fosdem-2024/cfp) website. This has
```
Comment (heliocola) [inline index.md:33]:
```suggestion
Head to the [FOSDEM 2024 Pretalx website](https://pretalx.fosdem.org/fosdem-2024/cfp). This has
```

**Review by heliocola — APPROVED** [review]

**Pull request merged**